_posts/2021-07-28-how-to-update-to-cis-14.adoc

```diff
@@ -0,0 +1,349 @@
+"Updating" section of "How to use the Gruntwork Infrastructure as Code Library"] for instructions on how to update the
+versions in your code.
+
+[.exceptional]
+IMPORTANT: Gruntwork follows
+https://gruntwork.io/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/#versioning[semantic
+versioning]. For any pre-1.0 modules, this means that version updates to the minor version are considered backward
+incompatible releases for any version updates before the 1.0.0 release. Make sure to read the release notes for the
+relevant modules any time you are updating minor versions! Note that you will want to read the release notes for each
+minor version that is updated (e.g., if you are going from v0.5.x to v0.9.x, you will want to read the notes for v0.6.0,
+v0.7.0, v0.8.0, and v0.9.0 to get the full list of backward incompatible updates).
+
+The following table provides a summary of all the relevant Gruntwork AWS modules and the respective versions that are
+compatible with CIS AWS v1.4.0:
+
+[[compatibility_table]]
+[cols="1,1h,1"]
+|===
+|Gruntwork Repo |Minimum version with CIS AWS v1.4.0 support |Corresponding CIS AWS v1.4.0 recommendations
+
+|terraform-aws-security
+|https://github.com/gruntwork-io/terraform-aws-security/releases/tag/v0.54.0[v0.54.0]
+|1.12, 2.1.3, 2.1.5
+|terraform-aws-cis-service-catalog
+|https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.27.0[v0.27.0]
+|2.1.4, 4.1
+
+|===
+
+=== Step 2: Update the Account Baseline modules
+
+Next, you will need to update the account baseline (landing zone) modules to the version compatible
+with v1.4.0 of the CIS AWS Foundations Benchmark. We strongly recommend upgrading straight to at least v0.27.0,
+instead of each minor version incrementally. This saves a lot of time due to the performance enhancements in
+recent versions of the account baseline modules and you can still complete the required migration steps before
+running Terraform/Terragrunt `apply`.
+
+NOTE: This guide assumes you are using at least v0.22.0 of the CIS Service Catalog repo.
+
+The account baseline modules had three breaking changes between versions v0.22.0 and v0.27.0. We must manually run
+these migration steps before updating the module versions.
+
+1. https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.23.0[v0.23.0]: Refactored the
+SecurityHub module to remove a Python script that managed invitations between the AWS accounts. It's necessary to run a
+state migration to manage the invitations with Terraform.
+2. https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.24.0[v0.24.0]: This release introduces MFA Delete. You will need to follow the migration guide to ensure all S3 buckets are properly secured. Note: It is unlikely you will need to perform this step on the AWS root account as they typically don't contain S3 buckets. Please ensure you migrate all other AWS accounts.
```

Comment:
NIT: If you use `account-baseline-root`, you'll likely have the Terraform state bucket in root.